asreview/data/base.py

```diff
@@ -362,14 +396,16 @@ def to_file(self, fp, labels=None, ranking=None):
         ranking: list, numpy.ndarray
             Optionally, dataframe rows can be reordered.
         """
+        df = self.to_dataframe(labels=labels, ranking=ranking)
+
         if Path(fp).suffix in [".csv", ".CSV"]:
```

Comment:
This should be based on the available entry points isn't it?

Reply:
Like in from_file

Reply:
Should `export_to_string` function in `project.py` be integrated into `to_file`?